components/testing/COMPONENT_FPGA_CI_TEST_SHIELD/MbedTester.h

```diff
@@ -0,0 +1,880 @@
+/*
+ * Copyright (c) 2019, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef MBED_TESTER_H
+#define MBED_TESTER_H
+
+#include "DynamicPinList.h"
+#include "platform/FileHandle.h"
+#include "platform/Callback.h"
+#include "drivers/DigitalInOut.h"
+
+class MbedTester {
```

Comment:
would it be worth documenting this class  (for instance as SPI class has some example and description). Isn't this the most important object for this shield?

Reply:
I added some documentation and example code.